### PR TITLE
chore: add more content and information to readme file

### LINKED ISSUE #1: enhance documentation
The readme.md file should be improved in order to provide helpful information

### PR DESCRIPTION
**Pull request by LeMollinger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,31 @@
...
+
+## Description
+
+This project contains a "one-line" script for enabling the installation of dependencies with the dedicated avengineers repository [bootstrap](https://github.com/avengineers/bootstrap).
+It is designed to simplify the process by using a Powershell script to clone the repository.
+This will then be used for setting up your environment by installing different dependencies.
+
+## How to use it
+
+Running `install.ps1` will clone the above mentioned repository into your current working directory.
```
Comment (xxthunder) [inline README.md]:
```suggestion
Running `install.ps1` will clone the above mentioned `bootstrap` repository to a directory `.bootstrap` in your current working directory.

```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,31 @@
...
+One can also utilize it by invoking a web-request (e.g. curl) to the url which is tied to the script's source.
+See the following example for Powershell:
+
+```powershell
+Invoke-RestMethod https://raw.githubusercontent.com/avengineers/bootstrap-installer/v1.6.0/install.ps1 | Invoke-Expression
+```
```
Comment (xxthunder) [inline README.md]:
```suggestion
```powershell
Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser
Invoke-RestMethod -Uri https://raw.githubusercontent.com/avengineers/bootstrap-installer/v1.6.0/install.ps1 | Invoke-Expression
```

```

Comment (xxthunder) [reply README.md]: Took that from here: https://scoop.sh/

File: README.md [hunk 3]
```diff
@@ -1,2 +1,31 @@
...
+
+Contributions are welcome. Please fork this repository and create a pull request if you have something you want to add or change.
+
+## License
+
+This project is licensed under the MIT License - see the LICENSE.md file for details.
+
+## Contact Information
+
+For any queries, please contact us via email.
```
Comment (xxthunder) [inline README.md]:
```suggestion
For any queries, please raise an issue in this repository.

```

**Review by xxthunder — CHANGES_REQUESTED** [review]
🦆

File: README.md (hunk 1 above)
Comment (LeMollinger) [reply README.md]: I incorporated this suggestion into the latest push

File: README.md (hunk 2 above)
Comment (LeMollinger) [reply README.md]: I incorporated this suggestion into the latest push

File: README.md (hunk 3 above)
Comment (LeMollinger) [reply README.md]: I incorporated this suggestion into the latest push

**Review by xxthunder — APPROVED** [review]
🚀

**Pull request merged**